components/membership/registration-impl/src/test/kotlin/net/corda/membership/impl/registration/dynamic/mgm/ExpirationProcessorTest.kt

```diff
@@ -244,6 +246,40 @@ class ExpirationProcessorTest {
             verify(publisher).start()
         }
 
+        // test required for 5.0 to 5.1 platform upgrade
+        @Test
+        fun `use hard-coded defaults until newest version of config is received via reconciliation`() {
+            val dummyTime = 100L
+            whenever(membershipConfig.getLong(MAX_DURATION_BETWEEN_EXPIRED_REGISTRATION_REQUESTS_POLLS))
+                .thenThrow(ConfigException.Missing("")).thenReturn(dummyTime)
+            whenever(membershipConfig.getLong(EXPIRATION_DATE_FOR_REGISTRATION_REQUESTS))
+                .thenThrow(ConfigException.Missing("")).thenReturn(dummyTime)
+
+            val capturedTimeframes = argumentCaptor<Long>()
+            val capturedEvents = argumentCaptor<(String) -> TimerEvent>()
+
+            handler.firstValue.processEvent(configChangedEvent, coordinator)
+            // new config received
+            handler.firstValue.processEvent(configChangedEvent, coordinator)
+
+            verify(coordinator, times(2)).setTimer(any(), capturedTimeframes.capture(), capturedEvents.capture())
+
+            SoftAssertions.assertSoftly {
+                it.assertThat(capturedTimeframes.firstValue).isLessThanOrEqualTo(TimeUnit.MINUTES.toMillis(180))
+                it.assertThat(capturedTimeframes.secondValue).isLessThanOrEqualTo(TimeUnit.MINUTES.toMillis(dummyTime))
+
+                val lambda = capturedEvents.firstValue.invoke("")
```

Comment:
Maybe some name like this would be better? If I'm reading it right this value isn't a lambda
```suggestion
                val declineEvent = capturedEvents.firstValue.invoke("")
```

Reply:
done in [5759425](https://github.com/corda/corda-runtime-os/pull/4126/commits/5759425a69bee48824ca7724fff6528f426f050f)